packages/scripts/utils/cli.js

```diff
@@ -0,0 +1,85 @@
+/**
```

Comment:
Checked that this file extracts the functions literally from `utils/index.js` with no changes.

Reply:
Yes, I wanted to have `index.js` with mostly re-exports.